.github/workflows/ci-deploy-snapshots.yml

```diff
@@ -0,0 +1,42 @@
+name: Quarkus CI
+
+on:
+  schedule:
+    - cron: '0 1 * * *'
+
+env:
+  LANG: en_US.UTF-8
+jobs:
+  ci-sanity-check:
+    name: "CI Sanity Check"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build
+        run: sleep 30
+  build-and-deploy:
+    name: "Build and deploy"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: master
+      - uses: n1hility/cancel-previous-runs@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Reclaim Disk Space
+        run: .github/ci-prerequisites.sh
+      - name: Set up JDK 11
+        # Uses sha for added security since tags can be updated
+        uses: joschi/setup-jdk@b9cc6eabf7e7e3889766b5cee486f874c9e1bd2d
+        with:
+          java-version: 11
+      - name: Build-And-Deploy
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_API_TOKEN }}
+          SERVER_USERNAME: ${{ secrets.SERVER_USERNAME }}
+          SERVER_PASSWORD: ${{ secrets.SERVER_PASSWORD }}
+        run: |
+          mvn -e -B --settings .github/mvn-settings.xml \
+            -DskipITs -Dno-format -Dinvoker.skip=true \
+            -DretryFailedDeploymentCount=10 \
+            clean deploy
```

Comment:
I think you also want to `-DskipTests`?

Reply:
Or `-Dmaven.test.skip` to skip test compilation also

Reply:
@gsmet wanted to run some tests.